lib/fuse_loop_mt.c

```diff
@@ -300,18 +301,21 @@ static void fuse_join_worker(struct fuse_mt *mt, struct fuse_worker *w)
 	free(w);
 }
 
-int fuse_session_loop_mt(struct fuse_session *se, int clone_fd)
+int fuse_session_loop_mt_config(struct fuse_session *se, struct fuse_loop_config *config);
+FUSE_SYMVER(".symver fuse_session_loop_mt_config,fuse_session_loop_mt@@FUSE_3.2");
+int fuse_session_loop_mt_config(struct fuse_session *se, struct fuse_loop_config *config)
```

Comment:
Please use `_fuse32` and `fuse31` suffixes instead of `_config`. That keeps it consistent with the naming of `fuse_new()`.